pkg/config/validation/validation.go

```diff
@@ -1469,9 +1481,14 @@ func ValidateProxyConfig(config *meshconfig.ProxyConfig) (errs error) {
 	}
 
 	if tracer := config.GetTracing().GetTlsSettings(); tracer != nil {
-		if err := validateTLS(tracer); err != nil {
+		v := validateTLS(tracer)
+		err, warning := v.Unwrap()
+		if err != nil {
 			errs = multierror.Append(errs, multierror.Prefix(err, "invalid tracing TLS config:"))
 		}
+		if warning != nil {
+			scope.Warnf("invalid tracing TLS config: %v", warning)
```

Comment:
Why are we logging rather than returning the warning?

Reply:
This is in meshconfig, which is not validated by webhook